### ISSUE TITLE
Different service and task definition names

### ISSUE THREAD

**Issue by Tensho** [aws/amazon-ecs-cli#435]
As far as I noticed ECS CLI allows to specify `--project-name` option to `ecs-cli compose` command and override default (current directory name) name for **both** – service and task definition. Is it possible somehow to distinguish name for the service and task definition at the moment?
```
$ ecs-cli compose --project-name abc --verbose service up
...
INFO[0001] Using ECS task definition                     TaskDefinition="abc:1"
INFO[0001] Created an ECS service                        service=abc taskDefinition="abc:1"
DEBU[0001] Updated ECS service                           count=1 service=abc
INFO[0001] Updated ECS service successfully              desiredCount=1 serviceName=abc
...
```
Intention of the option name is pretty clear – specify project name and ECS CLI will handle proper names for all needed components for you. But it would be nice to distinguish names somehow.

Please, could you also clarify the difference between `services` declarations across YAML configuration files:
```
# docker-compose.yml
version: '2'

services:
  <SERVICE_NAME_A>:
    ...
  <SERVICE_NAME_B>:
    ...
```
```
# ecs-params.yml
version: 1

task_definition:
  ...
  services:
    <SERVICE_NAME_C>:
       essential: true
    <SERVICE_NAME_D>:
       essential: true
```
What are relations between all that references?

BTW, you have slightly different actual naming description in cli and documentation. Everywhere in documentation (GitHub and AWS Docs) there is `service-` prefix appears before actual project name in the service name.

**Comment by SoManyHs** [comment]
Hi @Tensho,

Sorry for the delay. Unfortunately, the CLI does not currently provide a way to assign a different name to an ECS service and its Task Definition. If you really wanted to distinguish them, you could edit the Task definition in the AWS console manually. This would actually create a new Task Definition revision, which you'd then have to attach to your service by updating the service.

For your question about the two config files: The `services` field in `docker-compose.yml` corresponds directly to the `services` in `ecs-params.yml`. The fields from both are merged into an [ECS Container Definition](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-containerdefinitions.html). Currently the only field for a service from `ecs-params.yml`

As for the documentation, there is a fix for our README in #429. We'll let our docs team know about the discrepancy in the online docs. Thanks for pointing that out.

**Comment by Tensho** [comment]
Thank you for your answer, @SoManyHs 🙇
